packages/colorpickers/src/styled/ColorPicker/StyledAlpha.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { Range } from '@zendeskgarden/react-forms';
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { colorStyles, thumbStyles, trackStyles, trackLowerStyles } from './StyledHue';
+
+const COMPONENT_ID = 'colorpickers.colorpicker_alpha';
+
+/**
+ * 1. Adjust spacing in IE11 to match other browsers
+ * 2. Provides height for the input so that the thumb shadow styles are not cut off in IE11.
+ */
+export const StyledAlpha = styled((Range as unknown) as 'input').attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
```

Comment:
ditto re: simplifying the TS here by using `as: 'input'` attrs

Reply:
This is intended to fix a TypeScript compilation issue so modifying the DOM tag wouldn't fix the compilation issue.

It seems like there are multiple versions of `@types/react` when a package uses `react-modals`. The `react-transitiongroup` also depends on `@types/react` and is duplicating it. Austin and I attempted to use `yarn workspaces` to hoist this dependency but ran into issues around the build process. It looks like type-casting is a suitable workaround for now.